master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -269,7 +269,8 @@ private[celeborn] class Master(
   }
 
   masterSource.addGauge(MasterSource.DEVICE_CELEBORN_FREE_CAPACITY) { () =>
-    statusSystem.workers.asScala.toList.map(_.totalActualUsableSpace()).sum
+    statusSystem.workers.asScala.toList.filter(statusSystem.isWorkerAvailable).map(
```

Comment:
This change is not suitable especially there is a gauge "DEVICE_CELEBORN_TOTAL_CAPACITY" use all workers.

Reply:
yes, I consider add a new metrics likes `DEVICE_CELEBORN_AVAILABLE_CAPACITY`.

But seems the free also means available and can not distinguish them.